libraries/node-core-library/src/RealNodeModulePath.ts

```diff
@@ -146,7 +150,17 @@ export class RealNodeModulePathResolver {
     // of an lstat call.
     const stat: nodeFs.Stats | undefined = this._fs.lstatSync(link);
     if (stat.isSymbolicLink()) {
-      return this._fs.readlinkSync(link, 'utf8');
+      const result: string = this._normalizeTrailingSlash(this._fs.readlinkSync(link, 'utf8'));
+      return result;
+    }
+  }
+
+  private _normalizeTrailingSlash(input: string): string {
```

Comment:
```suggestion
  private _trimTrailingSlash(input: string): string {
```

Reply:
It doesn't always remove it, e.g. if the path is `/` on POSIX or `C:\` or `\\?\C:\` on Windows.

Reply:
Method no longer present, so don't need to name it.